sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/dtt/AbstractDataAttributeAdapter.java

```diff
@@ -122,12 +149,20 @@ public boolean hasSameContentAs(T data) {
         return true;
     }
 
+    /**
+     * Gets DataTypeTemplateAdapter
+     * @return DataTypeTemplateAdapter object
+     */
     @Override
     public DataTypeTemplateAdapter getDataTypeTemplateAdapter() {
         return ((IDataTemplate)parentAdapter).getDataTypeTemplateAdapter();
     }
 
-
+    /**
+     * Updates DA Type Name
```

Comment:
Note that we should rename this method, it is about checking and updating. `check` is not meaningful